packages/utils/schema/user-profile.graphql

```diff
@@ -0,0 +1,111 @@
+type ProfileName
+  @createModel(accountRelation: SINGLE, description: "Your name") {
+  name: String! @string(maxLength: 150)
+}
+
+type ProfileAvatar
+  @createModel(
+    accountRelation: SINGLE
+    description: "An image and optional fallback images representing your avatar"
+  ) {
+  original: ImageMetadata!
+  alternatives: [ImageMetadata] @list(maxLength: 3)
+}
+
+type ProfileDescription
+  @createModel(
+    accountRelation: SINGLE
+    description: "A short profile bio / blurb / description"
+  ) {
+  description: String! @string(maxLength: 420)
+}
+
+type ProfileEmoji
+  @createModel(accountRelation: SINGLE, description: "Your spirit emoji") {
+  emoji: String! @string(maxLength: 2)
```

Comment:
I think there are combining emoji sequences that consist of up to 5 characters.

Why not just let them specify a list of emoji. If the circumstances call for a single emoji we can just take the head of the list, or one at random.

I think the structure I like the most is an associative array where descriptors are linked to a string of emoji. Like "`happy`" linking to "`💆🏿‍♀️🤓👨🏿‍🌾🤪🧞‍♀️`".

Sites could pick a default descriptor to load the emoji set from, or allow users to choose.

Ultimately, I'd like to be supporting both emoji and vector images that can be used to represent the user accurately and concisely in a 2D interface.

```suggestion
  emoji: DescribedEmojiString @list(maxLength: 69420)
```

Reply:
This is an interesting idea but IMO needlessly complicated. At least we should support the basic `string` representation.  Models should be as self-explanatory as possible if we want them to be used.

Reply:
We can always add this as an additional field here